examples/transformer/model.py

```diff
@@ -0,0 +1,195 @@
+from typing import Optional
+
+import torch
+from torch import nn
+
+import texar as tx
+
+
+class Transformer(nn.Module):
+    r"""A standalone sequence-to-sequence Transformer model.
+    TODO: Add detailed docstrings.
```

Comment:
Pls fix the TODO

Reply:
I did not change this part of the code. Could @haoransh help fixing this?